extensions/blocks/membership-button/edit.jsx

```diff
@@ -154,6 +154,20 @@ class MembershipsButtonEdit extends Component {
 		);
 	};
 
+	renderAmount = product => {
+		const amount = formatCurrency( parseFloat( product.price ), product.currency );
+		if ( product.interval === '1 month' ) {
+			return sprintf( __( '%s /month', 'jetpack' ), amount );
+		}
+		if ( product.interval === '1 year' ) {
+			return sprintf( __( '%s /year', 'jetpack' ), amount );
+		}
+		if ( product.interval === '1 year' ) {
+			return amount;
+		}
+		return sprintf( __( '%s /%s', 'jetpack' ), amount, product.interval );
```

Comment:
I am not sure I understand that final fallback; if we add more interval, wouldn't it then be rendered as "$50 / 1 week"?

Reply:
Yes, it would. It's just a crude fallback
- If we remember to update the schedule everywhere, we will add another check here and will be fine
- If we dont, "$50 / 1 week" is better than nothing.
We may be rolling out new schedules on select sites on demand - that is why we are not preparing to have them for everyone now - but thats just a conjecture at this point.